### PR TITLE
[14.0][ADD] partner_category_security_crm: New addon.

### PR DESCRIPTION
**Pull request by victoralmau**
New addon.

Related to https://github.com/OCA/partner-contact/pull/1328#issuecomment-1228267032

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT38576

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: partner_category_security_crm/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a glue module between "partner_category_securit"y and "crm" to mitigate the permissions to manage contact tags.
```
Comment (CarlosRoca13) [inline partner_category_security_crm/readme/DESCRIPTION.rst]:
```suggestion
This is a glue module between "partner_category_security" and "crm" to mitigate the permissions to manage contact tags.
```

**Review by CarlosRoca13 — APPROVED** [review]
LGTM, just a minor comment

**Comment by victoralmau** [comment]
Changes done.

**Comment by pedrobaeza** [comment]
Please check tests

**Comment by victoralmau** [comment]
> Please check tests

A few days ago a similar temporary error occurred. The error occurs specifically here https://github.com/OCA/partner-contact/runs/8064614950?check_suite_focus=true#step:8:921 because the api does not return information.

**Pull request merged**